File: Go/FizzBuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 100; i++ {
		if i == 100 {
			fmt.Printf("%d", i)
		} else if i%15 == 0 {
			fmt.Printf("FizzBuzz, ")
		} else if i%5 == 0 {
			fmt.Printf("Buzz, ")
		} else if i%3 == 0 {
			fmt.Printf("Fizz, ")
		} else {
			fmt.Printf("%d, ", i)
		}
	}
}

File: Perl/fizzbuzz.pl
#!/usr/bin/env perl
use strict;
use warnings;
for my $i ( 1 .. 100 ) {
    if ( $i % 3 == 0 && $i % 5 == 0 ) { print 'fizzbuzz' }
    elsif ( $i % 3 == 0 ) { print 'fizz' }
    elsif ( $i % 5 == 0 ) { print 'buzz' }
    else { print $i }
    print "\n";
}
